Docs front page overhaul (#2569)
canonical/ubuntu-pro-client: docs/howtoguides/enable_realtime_kernel.md

```

You'll need to reboot after installing to boot into the Real-time kernel.

## Notes
- Real-Time Kernel is not compatible with Livepatch. If you wish to use the

>   Real-Time Kernel but Livepatch is enabled, you will
>   [need to disable Livepatch](https://ubuntu.com/security/livepatch/docs/livepatch/how-to/disable)
>   first.

Reviewer: ```suggestion
  Real-Time Kernel but Livepatch is enabled, `pro` will prompt and offer to disable Livepatch first.
```

Author: Thanks - also spotted a stray "ubuntu advantage" on this page that somehow escaped.
